py/redrock/external/desi.py

```diff
@@ -111,7 +115,8 @@ def read_spectra(spectrafiles, targetids=None):
 
     return targets, meta
 
-def read_bricks(brickfiles, trueflux=False, targetids=None):
+
+def read_bricks(brickfiles, trueflux=False, targetids=None, spectrum_class=None):
```

Comment:
Similar to `read_spectra`: document use of `spectrum_class` and consider making `SimpleSpectrum` the default.